### ISSUE TITLE
[auth] verification reminders script dies in stage/prod

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa#2182]
I noticed verification reminders didn't go out when I put v1.143.3 canary into prod. Looking back I can reproduce in stage too. Tough part is, this is all that the script says about why:

```
{"Timestamp":1565726549406000000,
"Logger":"fxa-auth-server","Type":"verificationReminders.fatal","Severity":2,"Pid":18,
"EnvVersion":"2.0","Fields":{"err":"TypeError: Cannot read property 'host' of undefined"}}
```

**Comment by philbooth** [comment]
Ah, dammit. It's because I forgot to update the verification reminders script with the new arguments to `Mailer`.

Great catch, thanks @jrgm, fix coming shortly.

**Comment by philbooth** [comment]
(tests should have caught it of course, I'll make sure to update those too)

**Comment by philbooth** [comment]
(turns out there are no tests for that script, sigh)

**Comment by philbooth** [comment]
Fixed by #2183.
